Tests/Unit/Hooks/HookProviderTest.php

```diff
@@ -0,0 +1,248 @@
+<?php
+declare(strict_types = 1);
+
+namespace OliverKlee\Seminars\Tests\Unit\Hooks;
+
+use Nimut\TestingFramework\TestCase\UnitTestCase;
+use OliverKlee\Seminars\Hooks\HookProvider;
+use OliverKlee\Seminars\Hooks\Interfaces\Hook;
+use OliverKlee\Seminars\Tests\Unit\Fixtures\Hooks\TestingHookInterface;
+use OliverKlee\Seminars\Tests\Unit\Fixtures\Hooks\TestingHookImplementor;
+use OliverKlee\Seminars\Tests\Unit\Fixtures\Hooks\TestingHookImplementor2;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+/**
+ * Test case.
+ *
+ * @author Michael Kramer <m.kramer@mxp.de>
+ */
+class HookProviderTest extends UnitTestCase
+{
+    /**
+     * @var array
+     */
+    private $extConfBackup = [];
+
+    protected function setUp()
+    {
+        parent::setUp();
```

Comment:
For unit tests, the `parent::setUp()` call is not needed (as that method is empty).